### DISCUSSION TITLE
Button Loading

### DISCUSSION THREAD

**Discussion by DisabledMonke** [saadeghi/daisyui#1738] [Q&A] [answered]
Hi,
Is there any non complicated way to make the loading animation for "btn loading" faster?
I haven't found anything in the docs :(

**Comment by saadeghi** [comment, marked as answer]
That button `loading` class is going to change in daisyUI `3.0`  
But for now you can change the speed using custom CSS or using an ugly class like this:  
`[&.btn:before]:[animation-duration:.3s]`  
https://play.tailwindcss.com/DlrGGnPgW1

Let me know if you have any questions
